### PR TITLE
Add object-fit-images@3.0.1 w/ npm auto-update

### LINKED ISSUE #9885: [Request] Add object-fit-images
**Library name:** object-fit-images
**Git repository url:** https://github.com/bfred-it/object-fit-images/
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #9885

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/bfred-it/object-fit-images/
 * Official website (optional, not the repository): https://github.com/bfred-it/object-fit-images/
 * NPM package url (optional): https://www.npmjs.com/package/object-fit-images
 * License and its reference: [MIT](https://github.com/bfred-it/object-fit-images/blob/master/license)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 15
   - Count of stars: 565
   - Count of forks: 65
 * NPM download stats (optional):
   - Downloads in the last day: 495
   - Downloads in the last week: 2736
   - Downloads in the last month: 8955

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9885, cc @bfred-it

### CODE REVIEW

**Comment by extend1994** [comment]
@PeterDaveHello please help me review this pull request, thank you.

File: ajax/libs/object-fit-images/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "filename": "ofi.min.js",
+  "version": "3.0.1",
+  "description": "Polyfill object-fit and object-position on images on IE9, IE10, IE11, Edge, Safari, ...",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/bfred-it/object-fit-images.git"
+  },
+  "author": "Federico Brigante <bfred-it@users.noreply.github.com> (twitter.com/bfred_it)",
+  "homepage": "https://github.com/bfred-it/object-fit-images/",
```
Comment (PeterDaveHello) [inline ajax/libs/object-fit-images/package.json]: @extend1994 please remove "homepage" since it's the same as its git repo url.

**Comment by extend1994** [comment]
@PeterDaveHello I removed `homepage`, please help me review it, thank you.

**Comment by PeterDaveHello** [comment]
@extend1994 I think the reason to add old versions manually is not because the naming scheme, but the file/directory structure.

**Comment by extend1994** [comment]
@PeterDaveHello I modified the commit message, please help me review it, thank you.

**Comment by extend1994** [comment]
@PeterDaveHello the homepage was removed, please help me review it, thanks.

**Pull request merged**